book/src/backends/plonky3.md

```diff
@@ -0,0 +1,3 @@
+# Plonky3
+
+powdr supports [plonky3](https://github.com/Plonky3/Plonky3) with the Goldilocks field.
```

Comment:
Maybe mention that it is not sound yet?

Reply:
It's sound, updated with missing features